services/web/server/src/simcore_service_webserver/computation_api.py

```diff
@@ -65,13 +65,41 @@ async def _get_node_details(
         app, node_key, node_version
     )
     if not node_details:
-        log.error("Error could not find service %s:%s", node_key, node_version)
+        log.error(
+            "Error (while getting node details) could not find service %s:%s",
+            node_key,
+            node_version,
+        )
         raise web_exceptions.HTTPNotFound(
             reason=f"details of service {node_key}:{node_version} could not be found"
         )
     return node_details
 
 
+async def _get_node_extras(
+    node_key: str, node_version: str, app: web.Application
+) -> Dict:
+    """Returns the service_extras if possible otherwise None"""
+    if "file-picker" in node_key:
+        return None
+    if "frontend/nodes-group" in node_key:
+        return None
+    if "StimulationSelectivity" in node_key:
+        return None
```

Comment:
Not knowing the implementation details for this. The best I could do now was to refactor it in a cleaner way.

Reply:
what I mean here is at line 304 in the same file:
https://github.com/ITISFoundation/osparc-simcore/blob/58280a68e61538f55f7011d0eda29482598797b3/services/web/server/src/simcore_service_webserver/computation_api.py#L304-L305

Reply:
this might be also useful for the comments of @pcrespov  below

Reply:
so each node in the comp_task DB gets a "node_class" entry. all these services gets it set as FRONTEND, so you can filter using that instead of the service names

Reply:
I see now what you are meaning.